x/delayedack/eibc.go

```diff
@@ -124,6 +125,10 @@ func (im IBCMiddleware) createDemandOrderFromIBCPacket(fungibleTokenPacketData t
 		demandOrderDenom = trace.IBCDenom()
 		demandOrderRecipient = fungibleTokenPacketData.Sender // and who tried to send it (refund because it failed)
 	case commontypes.RollappPacket_ON_RECV:
+		bridgingFee := im.keeper.BridgingFee(ctx).MulInt(amt).TruncateInt()
+		if bridgingFee.GT(fee) {
```

Comment:
```suggestion
	case commontypes.RollappPacket_ON_RECV:
		bridgingFee := im.keeper.BridgingFee(ctx).MulInt(amt).TruncateInt()
		if bridgingFee.GT(fee) {
		// We check that the fee the fulfiller makes is at least as big as the bridging fee they will have to pay later
		// this is to improve UX and help fulfillers not lose money.
```

Reply:
Could you add a method to delayedack keeper to calculate the fee, given an amt
and DRY out with the usage here https://github.com/dymensionxyz/dymension/blob/627a9376cacca40c2d461b021520eeb578b972d8/x/bridging_fee/ibc_middleware.go#L84

Reply:
TBH
If there isn't a technical reason for this I'm not sure we should include it
It doesn't seem to me for delayedack to have awareness of bridging fee as a concept
And we should rely on frontend for these kind of UX wins

Reply:
It makes no sense to create a "losing" order for the market makers

Reply:
It does make sense from the code perspective though
It's arguably more code to maintain, for a problem which can be solved on the frontend level (before it becomes a TX) more easily

Reply:
I believe the general guideline of moving user UX/responsiblity off-chain makes sense in many cases and we should probably have this in mind. however as this is already implemented I'd suggest not to change it now and continue with current implementation.